main.go

```diff
@@ -103,6 +103,10 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "PipelineRun")
 		os.Exit(1)
 	}
+	if err = (&appstudiov1alpha1.ReleaseLink{}).SetupWebhookWithManager(mgr); err != nil {
```

Comment:
Could you add the variable to control whether we want webhooks or not here? https://github.com/redhat-appstudio/application-service/blob/main/main.go#L171

Not sure if it requires other changes, but seems like a good to have and it's not application-service specific as I've seen it in other operators.